crates/anstream/Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "anstream"
 version = "0.6.19"
-description = "A simple cross platform library for writing colored text to a terminal."
+description = "A simple cross platform library for writing (and auto-adapting) colored text to a terminal."
```

I feel like it works as a clarification after, rather than being parenthetical in the middle
```suggestion
description = "A simple cross platform library for writing colored text to a terminal, including adapting for the terminal's capabilities."
```

In looking over this though, I feel this can be confused with the ability to generate colored text and wonder if we should focus on this being about IO stream adapters that handle colored text appropriately for your terminal and environment.  Not sold on a specific wording for that though.

---

I feel like that wording might be confusing.  Something else I realized is that we put too much unimportant detail at the front and could lose people.

> IO stream adapters for writing colored text that will gracefully degrade according to your terminal's capabilities.

---

I think that's good. I would recommend the more layman term "downgrade" over "degrade"